toolkit/scripts/pkggen.mk

```diff
@@ -182,6 +183,8 @@ $(STATUS_FLAGS_DIR)/build-rpms.flag: $(cached_file) $(chroot_worker) $(go-schedu
 		$(if $(CONFIG_FILE),--base-dir="$(CONFIG_BASE_DIR)") \
 		$(if $(filter y,$(RUN_CHECK)),--run-check) \
 		$(if $(filter y,$(STOP_ON_PKG_FAIL)),--stop-on-failure) \
+		$(if $(filter n,$(USE_PACKAGE_BUILD_CACHE)),--no-cache) \
+		$(if $(filter n,$(CLEANUP_PACKAGE_BUILDS)),--no-cleanup) \
```

Comment:
Out of scope, but one thing I've noticed, that sill needs addressing when it comes to `CLEANUP_PACKAGE_BUILDS` : even with this switch `rpmbuild` will still automatically remove the `BUILD` directory, because we're using the `--rebuild` switch while running it (see: [here](http://ftp.rpm.org/max-rpm/s1-rpm-b-command-other-build-commands.html) and [here](https://www.man7.org/linux/man-pages/man8/rpmbuild.8.html)). We need that switch, but it would be nice, if there was a way to leave the build folder untouched. I haven't dug deep enough into the documentation to figure out what would be the best way to allow us that (`--nocleanup` switch only prevents `rpmbuild` from running the `%clean` section, but the build directory still gets removed), but in case you already know, it might be a good time to add it.

For this PR: I'd also mention in the documentation that the build directory will still get removed.

Reply:
I'll include in this pr a small tweak to `pkgworker` that leaves the buildroot intact on successful builds when `nocleanup` is set. There's a simple `--noclean` flag that rpmbuild respects.